src/Common/WellKnownTypeNames.cs

```diff
@@ -1,18 +1,16 @@
 ﻿namespace Moq.Analyzers.Common;
 
-#pragma warning disable ECS0200 // Consider using readonly instead of const for flexibility
-
 internal static class WellKnownTypeNames
 {
-    internal const string Moq = nameof(Moq);
-    internal const string MockName = "Mock";
-    internal const string MockBehavior = nameof(MockBehavior);
-    internal const string MockFactory = nameof(MockFactory);
-    internal const string MoqMock = $"{Moq}.{MockName}";
-    internal const string MoqMock1 = $"{MoqMock}`1";
-    internal const string MoqBehavior = $"{Moq}.{MockBehavior}";
-    internal const string MoqRepository = $"{Moq}.MockRepository";
-    internal const string As = nameof(As);
-    internal const string Create = nameof(Create);
-    internal const string Of = nameof(Of);
+    internal static readonly string Moq = nameof(Moq);
```

Comment:
The use of `nameof()` in these cases is wrong. These are aren't "true" constants like pi, but they are in the sense that renaming this field shouldn't change the value.

Reply:
The use in the code base is dual purpose: 

one is the namespace (which is probably as close as you're going to get to being constant), 

https://github.com/rjmurillo/moq.analyzers/blob/bbb213df094b4bf52314a5a066a9c775280ea013/src/Common/MoqMethodDescriptorBase.cs#L14

the other is for a type name

https://github.com/rjmurillo/moq.analyzers/blob/bbb213df094b4bf52314a5a066a9c775280ea013/src/Analyzers/ConstructorArgumentsShouldMatchAnalyzer.cs#L300

The convention in WellKnownTypeName.cs can be updated to make the use clearer and avoid tripping either rule, example:

```suggestion
    internal static readonly string MoqNamespace = "Moq";
    internal static readonly string MoqSymbolName = "Moq";
```

Reply:
I'm happy with a change like that

Reply:
In a4e7819b8b24adeb5e0eec7537b974f04c85b89f